selfdrive/locationd/laikad.py

```diff
@@ -396,7 +398,9 @@ def fetch_navs(self, t: GPSTime, block):
 
 
 def get_orbit_data(t: GPSTime, valid_const, auto_update, valid_ephem_types, cache_dir):
-  astro_dog = AstroDog(valid_const=valid_const, auto_update=auto_update, valid_ephem_types=valid_ephem_types, cache_dir=cache_dir)
+  warnings.filterwarnings("ignore", category=ResourceWarning, message="unclosed.*<zmq.*>")
```

Comment:
hmm, we don't want to just ignore these. you know where it's coming from?

Reply:
Kacper was looking into it. AFAIK it just comes from having an open zmq socket when running forking a process. 

```
import warnings
warnings.filterwarnings("default")
import zmq
import os

zctx = zmq.Context()
sock = zctx.socket(zmq.PUSH)
#sock.close()

pid = os.fork()
```
Is there a solution except manually closing the socket before spawning a child, everywhere you imported cloudlog? Because that doesn't seem great.

Reply:
https://github.com/commaai/openpilot/issues/23666#issuecomment-1071002173